target/linux/kirkwood/image/Makefile

```diff
@@ -182,6 +182,15 @@ define Device/ctera_c200-v1
 endef
 TARGET_DEVICES += ctera_c200-v1
 
+define Device/dlink_dns320l
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DNS-320L
+  DEVICE_PACKAGES := kmod-ata-marvell-sata kmod-fs-ext4 kmod-gpio-button-hotplug \
+	kmod-usb-storage kmod-usb-ledtrig-usbport dns320l-mcu
+  SUPPORTED_DEVICES += dns320l
```

Comment:
Why is this needed?

Reply:
Which part?

Reply:
`SUPPORTED_DEVICES` line. In most cases is added because backward compatibility is required.